sdk/core/ts-http-runtime/src/util/userAgentPlatform-browser.mts

```diff
@@ -31,24 +31,13 @@ interface NavigatorEx extends Navigator {
 function getBrandVersionString(
```

Comment:
We might want to get brand from `localNavigator.userAgentData.brands` since it's low entropy and unlikely to identify users, according to the MDN doc: https://developer.mozilla.org/en-US/docs/Web/API/NavigatorUAData/getHighEntropyValues

> Note: The terms high entropy and low entropy refer to the amount of information these values reveal about the browser. The values returned as properties are deemed low entropy, and unlikely to identify a user. The values returned by getHighEntropyValues() could potentially reveal more information. These values are therefore retrieved via a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise), allowing time for the browser to request user permission, or make other checks.

The "request user permission" part is a little concerning

Reply:
It's the same value as to whether you get it from the regular values versus high entropy values.  The only thing we get from the high entropy values is the OS version, which is helpful to know.